env/elastica/Docker70

```diff
@@ -34,5 +34,7 @@ ENV PATH=/root/composer/vendor/bin:$PATH
 
 COPY composer.json /root/composer/
 
+RUN composer global require --no-update friendsofphp/php-cs-fixer:^2.0
```

Comment:
I really don't like it putting it here, but I understand it could be the only solution until php-cs-fixer is released for PHP 7.3